tests/rcpsp_multiskill/test_mspsp_imopse_cpsat.py

```diff
@@ -0,0 +1,32 @@
+#  Copyright (c) 2024 AIRBUS and its affiliates.
+#  This source code is licensed under the MIT license found in the
+#  LICENSE file in the root directory of this source tree.
+#  In this example script we're parsing dzn files that are retrieved from https://github.com/youngkd/MSPSP-InstLib
+#  And run CP solver with different mzn models.
+
+import logging
+
+from discrete_optimization.generic_tools.cp_tools import ParametersCP
+from discrete_optimization.rcpsp_multiskill.rcpsp_multiskill_parser import (
+    get_data_available,
+    parse_file,
+)
+from discrete_optimization.rcpsp_multiskill.solvers.cpsat_msrcpsp_solver import (
+    CPSatMSRCPSPSolver,
+)
+
+
+def test_imopse_cpsat():
+    file = [f for f in get_data_available() if "100_5_64_9.def" in f][0]
+    model, _ = parse_file(file, max_horizon=1000)
+    cp_model = CPSatMSRCPSPSolver(
+        problem=model,
+    )
+    cp_model.init_model(
+        one_worker_per_task=True,
+    )
+    cp_model.cp_model.Minimize(cp_model.variables["makespan"])
```

Comment:
Same. Why this line?

Reply:
it's not needed anymore but i think i was testing other objectives ^^, will maybe remove